### DISCUSSION TITLE
What did you pack the assembly with?

### DISCUSSION THREAD

**Discussion by DestroyerDarkNess** [NickeManarin/ScreenToGif#1323] [General]
Hi, sorry to ask this way, but I'm curious.

I noticed that the main assembly is packaged with a native layer and that you also bundled all the libraries with the .exe .

What did you use? (again, sorry for asking this way)

**Comment by NickeManarin** [comment]
Hi, for questions please create "discussions", not issues :)

I simply use the publish within Visual Studio, with a few custom publish profiles.

**Reply by DestroyerDarkNess** [reply]
Thanks for responding so quickly. Do you happen to have any documentation on this? I didn't know that feature.
